crates/sui-core/src/authority_server.rs

```diff
@@ -413,7 +413,7 @@ impl ValidatorService {
         // CRITICAL! Validators should never sign an external system transaction.
         fp_ensure!(
             !certificate.is_system_tx(),
-            SuiError::InvalidSystemTransaction.into()
+            dbg!(SuiError::InvalidSystemTransaction.into())
```

Comment:
I'm now less sure that these should be removed -- but usually we don't keep `dbg!`s around, do we?

Reply:
I'll remove them